src/ui/OmniboxResultList/OmniboxResultList.ts

```diff
@@ -334,6 +335,14 @@ export class OmniboxResultList extends ResultList implements IComponentBindings
     );
     window.location.href = result.clickUri;
   }
+
+  protected initResultContainer(): ResultContainer {
```

Comment:
This, except one line, is pretty much what the initResultContainer from ResultList does, maybe you can refactor it so it doesn't repeat the code too much 😉

Reply:
hmmmmm 🤔 
I think I'll keep initResultContainer private in the ResultList but create a protected function to add the element to the dom and overwrite it OmniboxResultList to do nothing I'll have explain why the method doesn't do anything because it is a little weird to overwrite a function and having it empty